_data/sites.json

```
@@ -2794,6 +2794,15 @@
         ]
     },
 
+    {
+        "name": "BOOTH",
+        "url": "https://accounts.booth.pm/settings/delete",
+        "difficulty": "medium",
+        "notes": "Go to your profile page and the \"Delete Account\" button. If you did not set a password, it will ask you to create one before you can delete your account.",
```

```suggestion
        "difficulty": "easy",
        "notes": "Go to your profile page and the \"Delete Account\" button. If you did not set a password, it will ask you to create one before you can delete your account.",
```

Sounds `easy` from the description